modules/create-object-collections.adoc

```diff
@@ -0,0 +1,66 @@
+//
+// * operating/create-use-collections.adoc
+:_content-type: PROCEDURE
+[id="creating-object-collections_{context}"]
+= Creating object collections
+
+To access collections, click *Platform Configuration* -> *Collections*. The page displays a list of currently-configured collections. You can perform the following actions:
+
+* Search for collections by entering text in the *Search by name* field, and then press *->*. 
+* Click on a collection in the collection list to view the collection in read-only mode.
+* Click {kebab} for an existing collection to edit, clone, or delete it.
++
+[NOTE]
+====
+You cannot delete a collection that is actively used in {product-title-short}. ##I tried testing this, but I didn't get an error message and it allowed me to delete a collection that I'd used in a vulnerability report. Using stagingdb.demo.stackrox.com##
+====
+
+.Procedure
+To create an object collection:
+
+. In the Collections page, click *Create collection*.
+. Enter the name and description for the collection.
+. In the *Collection rules* section, you must define the rules for the collection. The results of your choices in this section are displayed in the *Collection results* live preview panel. Click *Hide results* to remove this panel from display. 
+. Optional: Add additional collections to a collection. Grouping collections and adding them to other collections can be useful if you want to create small collections based on deployments, then reuse and combine those smaller collections into larger, hierarchical collections. To add additional collections to a new collection that you are creating:
+.. In the *Attached collections* section, select a collection from the *Available collections* list or enter text in the *Filter by name* field and press *->*. Clicking the name of the collection displays information about the collection, such as the name and rules for the collection. After viewing collection information, close the window to return to the *Available collections* list.
+.. Click *+Attach*. Collections you attach are listed in the *Attached collections* section.
+. Click *Save*. 
+
+[id="creating-collection-rules"]
+== Creating collection rules
+
+Use the following guidelines when creating collection rules:
+
+* You must configure at least one rule or attach another collection to the new one that you are creating. 
++
+[NOTE]
+====
+For {product-title-short} release 3.74, collections are supported only for deployments, so you must configure a deployment object.
+====
+*  You first configure rules to include all or specific deployments, and then you can further filter the collection to include objects meeting certain criteria in namespaces and clusters.
+
+.Procedure
+
+. In the *Deployments* section, choose one of the following options from the drop-down list:
+
+* *All deployments*: Includes all deployments in the collection. If you choose this option, you must then filter the collection using namespaces or clusters.
```

Comment:
`... you must then filter the collection using namespaces, clusters, or attaching another collection.` Or something along those lines